src/components/organisms/Sider/Sider.styled.tsx

```diff
@@ -114,3 +116,33 @@ export const StyledOtherItem = styled.div`
     }
   }
 `;
+
+export const DropdownListItem = styled.li`
+  display: flex;
+  align-items: center;
+
+  height: 50px;
+
+  cursor: pointer;
+  transition: 0.3s;
+
+  a {
+    flex: 1;
+
+    padding: 14px;
+  }
+
+  &:hover {
```

Comment:
It should be `:hover` on `a` (link), not on the list element, as you may interact with the link, not `li`.